03/app.js

```diff
@@ -0,0 +1,12 @@
+const buttons = document.querySelectorAll("button")
+
+buttons.forEach((button) => {
+  button.addEventListener(
+    "click",
+    function () {
```

Comment:
Warto pamiętać, że w ten sposób mamy 3 różne funkcje. Jeśli zadeklarujemy wcześniej funkcje  i tutaj ją przekażemy to będzie tylko jedna :)